src/rpc/blockchain.cpp

```diff
@@ -1549,23 +1564,64 @@ RPCHelpMan getblockchaininfo()
+                                    {RPCResult::Type::NUM_TIME, "start_time", "the minimum median time past of a block at which the bit gains its meaning"},
+                                    {RPCResult::Type::NUM_TIME, "timeout", "the median time past of a block at which the deployment is considered failed if not yet locked in"},
+                                    {RPCResult::Type::NUM, "since", "height of the first block to which the status applies"},
+                                    {RPCResult::Type::NUM, "min_activation_height", "minimum height of blocks for which the rules may be enforced"},
+                                    {RPCResult::Type::OBJ, "statistics", /*optional=*/ true, "numeric statistics about signalling for a softfork (only for \"started\" and \"locked_in\" status)",
+                                    {
+                                        {RPCResult::Type::NUM, "period", "the length in blocks of the signalling period"},
+                                        {RPCResult::Type::NUM, "threshold", /*optional=*/ true, "the number of blocks with the version bit set required to activate the feature (only for \"started\" status)"},
+                                        {RPCResult::Type::NUM, "elapsed", "the number of blocks elapsed since the beginning of the current period"},
+                                        {RPCResult::Type::NUM, "count", "the number of blocks with the version bit set in the current period"},
+                                        {RPCResult::Type::BOOL, "possible", /*optional=*/ true, "returns false if there are not enough blocks left in this period to pass activation threshold (only for \"started\" status)"},
+                                    }},
+                                }},
+                                {RPCResult::Type::NUM, "height", /*optional=*/ true, "height of the first block which the rules are or will be enforced (only for \"buried\" type, or \"bip9\" type with \"active\" status)"},
+                                {RPCResult::Type::BOOL, "active", "true if the rules are enforced for the mempool and the next block"},
+                            }},
+        RPCExamples{ HelpExampleCli("getdeploymentinfo", "") + HelpExampleRpc("getdeploymentinfo", "") },
+        [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+        {
+            ChainstateManager& chainman = EnsureAnyChainman(request.context);
+            LOCK(cs_main);
+            CChainState& active_chainstate = chainman.ActiveChainstate();
+
+            const CBlockIndex* tip = active_chainstate.m_chain.Tip();
+            CHECK_NONFATAL(tip);
+            const Consensus::Params& consensusParams = Params().GetConsensus();
+
+            UniValue deploymentinfo(UniValue::VOBJ);
+            deploymentinfo.pushKV("deployments", DeploymentInfo(tip, consensusParams));
```

Comment:
Nesting everything under `deployments` seems useless.

Reply:
It separates out `height` and `hash` which are introduced in the following commit.